util/plugins/eu.esdihumboldt.util.groovy.sandbox/src/eu/esdihumboldt/util/groovy/sandbox/internal/RestrictiveGroovyInterceptor.java

```diff
@@ -162,6 +162,7 @@ public class RestrictiveGroovyInterceptor extends GroovyInterceptor {
 		allowedClasses.add(HashSet.class);
 		allowedClasses.add(Collections.class);
 		allowedClasses.add(TreeMap.class);
+		allowedClasses.add(Throwable.class);
```

Comment:
@JohannaOtt Please add another section for this. Adding it to the section "Some more collections" doesn't seem appropriate. E.g. like this:

```
// Exception classes
allowedClasses.add(Throwable.class);
```

Reply:
Done